src/universal/subscriptions/NotificationsAddedSubscription.js

```diff
@@ -58,6 +61,10 @@ const subscription = graphql`
           project {
             id
             content
+            teamMember {
```

Comment:
@mattkrick I went ahead and added this too for project notifications, since it was added to the socket route.